examples/fine-tuning/create_dataset.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ "$#" -ne 2 ] || ! [ -d "$2" ]; then
+  echo "Usage: $0 <NEW_DATASET_DIR> <MNIST_DIR>" >&2
+  exit 1
+fi
+
+NEW_DATASET_DIR=$1
+MNIST_DIR=$2
+
+# create new dataset directories
+mkdir -p $NEW_DATASET_DIR/odd
+mkdir -p $NEW_DATASET_DIR/even
+
+# create symbolic links
+for digit in 0 2 4 6 8
+   do cp $MNIST_DIR/$digit/*.png $NEW_DATASET_DIR/even/
```

Comment:
Why do you say that shuffling wasn't working across folders? It looks like it's working to me ...

Are you looking at the file that you get when clicking on "Input File (before shuffling)"? As it says, that's the list of files before shuffling. You can look at the actual order in the database with the "Explore the db" tool.